app/uk/gov/hmrc/economiccrimelevyaccount/config/AppConfig.scala

```diff
@@ -37,6 +37,7 @@ class AppConfig @Inject() (configuration: Configuration, servicesConfig: Service
   val signInUrl: String       = configuration.get[String]("urls.signIn")
   val signOutUrl: String      = configuration.get[String]("urls.signOut")
   val registrationUrl: String = configuration.get[String]("urls.registration")
+  val submitReturn: String    = configuration.get[String]("urls.submitReturn")
```

Comment:
Suggest renaming this as suggested to match the naming convention of other Url's

```suggestion
  val returnsUrl: String      = configuration.get[String]("urls.submitReturn")
```